cms/djangoapps/contentstore/models.py

```diff
@@ -175,6 +188,29 @@ def filter_links(
         )
         if ready_to_sync is not None:
             result = result.filter(ready_to_sync=ready_to_sync)
+
+        # Handle top-level parents logic
+        if use_top_level_parents is not None:
+            # Get objects without top_level_parent_usage_key
+            objects_without_top_level = result.filter(top_level_parent_usage_key=UsageKeyField.Empty)
```

Comment:
Above this variable is called `non_top_level_objects` and here it's called `objects_without_top_level`.
Same for `non_top_level_ids` and `ids_without_top_level`.
Would it make sense to make those consistent? And maybe adding a comment in each stating that this logic is very similar to what's in that other function, but pointing out the differences (if any)

Reply:
Updated https://github.com/openedx/edx-platform/pull/37076/commits/e77a67f214681adbbb09a5a58e07b8a1c2800e57